tensorflow/compiler/xla/service/gpu/llvm_gpu_backend/gpu_backend_lib.cc

```diff
@@ -726,6 +804,21 @@ StatusOr<std::vector<uint8>> CompileToHsaco(
 
   std::vector<uint8> hsaco;
   std::unique_ptr<llvm::TargetMachine> target_machine;
+  std::string ir_str;
+  llvm::raw_string_ostream stream(ir_str);
+  stream << *module;
+  std::string str = stream.str();
```

Comment:
You can use `ir_str`, no need to copy it into `str`.

Reply:
OK